rally_ovs/plugins/ovs/scenarios/ovn_fake_multinode.py

```diff
@@ -418,6 +418,11 @@ def setup_switch_per_node(self, fake_multinode_args = {},
                               lport_create_args = {},
                               port_bind_args = {},
                               create_mgmt_port = True):
+        # TODO: figure out how to not reload the context?
```

Comment:
@putnopvut This is the hack I was talking about.

Reply:
Can you go into a bit more detail about what's going on here? I've been away from this for long enough that I don't even know what it means to reload the context.

Reply:
`self.context` gets populated once before any iteration of the scenario is run, in `class OvnNorthboundContext.setup()`.  Then, if I'm not wrong, each scenario gets a copy of the context.  However, because we're updating load balancers in each iteration, ideally we would have to update the original context, such that upcoming iterations have the updated set of load balancers.

The rally runner doesn't allow that because iterations might be run in parallel, I presume.

To work around it I explicitly updated self.context["ovn-nb-lbs"] with the results of `ovn_nbctl.lb_list()`.